R/label_variables.R

```diff
@@ -0,0 +1,45 @@
+#' @title Label FMI variables
+#' @description `label_variables()` adds labels for the FMI observation variables.
+#'
+#'
+#' @param x The FMI dataset, which variables you want labeled.
+#' @param replcace A logical for whether to replace the variable values with the full label.
+#' Default is `FALSE`, which adds a new column for the labels.
+#'
+#' @import dplyr
```

Comment:
I think it would be better to use `@importFrom` here (and potentially elsewhere) so the imports wouldn't be too cluttered